ROADMAP MAR18-SEP18.md

```diff
@@ -13,46 +13,60 @@ Here are my current thoughts on a roadmap for our overall engineering organziati
 
-* Templates for static pages.  One of the biggest bottlenecks over the past year has been React - it can be a difficult framework to get started with, especially when someone is brand new to web development.  Rather than attempt to rewrite our front end in a different framework, I’d like to focus on making our particuarly implementation of React more beginner friendly.  Adding in static content is a great way for a beginner to get started with contributing - let’s make a template that can be copy and pasted for any new static page.  Yes, we might violate the DRY rule a bit here, but I think this is a worthy exception and necessary to our mission.  We need the process of using this template very well documented.  Any new copy for the front end should go in a Github issue with a link to this guide, and these issues should be reserved for first time contributions whenever possible.
+* Templates for static pages.  One of the biggest bottlenecks over the past year has been React - it can be a difficult framework to get started with, especially when someone is brand new to web development.  Rather than attempt to rewrite our front end in a different framework, I’d like to focus on making our particuarly implementation of React more beginner friendly.  Adding in static content is a great way for a beginner to get started with contributing - let’s make a template that can be copy and pasted for any new static page.  Yes, we might violate the DRY rule a bit here, but I think this is a worthy exception and necessary to our mission.  We need the process of using this template very well documented.  Any new copy for the front end should go in a Github issue with a link to this guide, and these issues should be reserved for first time contributions whenever possible. _does kyle's work negate this?_
 
-* Host docsify page on docs.operationcode.org - we need to have our docs in one easy to find and easy to navigate place, this is essential to making sure people can find the information they need quickly.  
+* ~~Host docsify page on docs.operationcode.org - we need to have our docs in one easy to find and easy to navigate place, this is essential to making sure people can find the information they need quickly.~~ _deprioritized, github will suffice for now -nell_
 
-* Landing Page ReDesign
+* *P1* Landing Page ReDesign _we need to communicate who we are and how you become a part of us, how to join. How do they know if we're legitimate? -davidm_
+ * Re-think and redo hero banner implementation
 
 * Introduce a drop-down item for the navbar.
 
-* Re-think and redo hero banner implementation
-
 * Make sure that all UI components have Jest snapshot tests and Storybook instances
+ * _in progress with kyle_
 
 * Document how to add a Jest test and a Storybook instance of a component
+ * _in progress with kyle_
 
 * Discuss moving shared components into UI Lib via NPM
+ * _check with kyle if this is still necessary -nell_
```

Comment:
I didn't know this was a plan. We can definitely do this, but since we only serve a single application rn, it doesn't seem necessary.